### PR TITLE
initial german translations

### PR DESCRIPTION
**Pull request by rschnapka**

### CODE REVIEW

File: asterisk_click2dial/i18n/de.po [hunk 1]
```diff
@@ -0,0 +1,656 @@
...
+#: code:addons/asterisk_click2dial/asterisk_click2dial.py:414
+#, python-format
+msgid "The '%s' for the user '%s' should only have ASCII caracters"
+msgstr "'%s' des Anwenders '%s' sollte nur aus ASCII-Zeichen bestehen"
+
+#. module: asterisk_click2dial
+#: code:addons/asterisk_click2dial/asterisk_click2dial.py:136
+#, python-format
+msgid "The '%s' should only have ASCII caracters for the Asterisk server '%s'"
+msgstr ""
```
Comment (pedrobaeza) [inline asterisk_click2dial/i18n/de.po]: Extra line feed

File: asterisk_click2dial/i18n/de.po [hunk 2]
```diff
@@ -0,0 +1,656 @@
...
+#. module: asterisk_click2dial
+#: field:res.users,variable:0
+msgid "User-specific Variable"
+msgstr "Anwenderspezifische Variable"
+
+#. module: asterisk_click2dial
+#: help:res.users,dial_suffix:0
+msgid "User-specific dial suffix such as aa=2wb for SCCP auto answer."
+msgstr ""
+"Anwenderspezifisches Wählsuffix, wie z. B. aa=2wb zum Auto-Abheben bei SCCP."
```
Comment (pedrobaeza) [inline asterisk_click2dial/i18n/de.po]: Extra line-feed

**Comment by yvaucher** [comment]
:+1:

**Comment by alexis-via** [comment]
:+1:

**Pull request merged**